src/cockpit/samples.py

```diff
@@ -121,21 +126,24 @@ def detect_cpu_sensors(self, hwmonid: int, name: str) -> None:
                 if name != 'cpu_thermal':
                     continue
 
-            self.sensors.append(sensor_path)
+            self.sensors.append(entry.path)
 
-    def sample(self, samples: Samples) -> None:
+    def sample(self, samples: Samples, hwmon_path='/sys/class/hwmon') -> None:
```

Comment:
No.  GitHub is rightly annoyed about this.

Use a constant defined at the toplevel and switch it out using `unittest.mock.patch`

Reply:
Hmm? What is this response too? Sure can do! Feels cleaner.